src/main/java/kubatech/tileentity/gregtech/multiblock/GT_MetaTileEntity_ExtremeIndustrialGreenhouse.java

```diff
@@ -625,15 +626,17 @@ public ItemStack transferStackInSlot(EntityPlayer aPlayer, int aSlotIndex) {
                 return super.transferStackInSlot(aPlayer, aSlotIndex);
             }
             if (mte.addCrop(aStack)) {
-                s.putStack(null);
+                if (aStack.stackSize == 0) s.putStack(null);
+                else s.putStack(aStack);
```

Comment:
Is this else clause needed?

Reply:
not sure

Reply:
On second thought, this makes sure change to stacksize will be announced, so it's ok